lib/node_modules/@stdlib/stats/base/dists/planck/pmf/README.md

```diff
@@ -0,0 +1,144 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2024 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# Probability Mass Function
+
+> Planck distribution [probability mass function][pmf] (PMF).
+
+<section class="intro">
+
+The [probability mass function][pmf] (PMF) for a planck random variable is defined as
```

Comment:
As `Planck` is a proper noun, it should be capitalized in all descriptions here and elsewhere.